production/scrapers/kentucky.R

```diff
@@ -10,7 +10,7 @@ kentucky_date_check <- function(x, date = Sys.Date()){
         {.[str_detect(., "(?i)as of")]} %>% 
         str_split("as of") %>% 
         unlist() %>% 
-        {.[str_detect(., "21")]}%>% 
+        {.[str_detect(., "\\d{1,2}-\\d{1,2}-\\d{1,2}")]} %>% 
```

Comment:
Recommended change:

```
    base_html %>%
        rvest::html_nodes("h6") %>%
        rvest::html_text() %>% 
        {.[str_detect(., "(?i)as of")]} %>% 
        str_split("as of") %>% 
        unlist() %>%
        .[[2]] %>%
        trimws() %>%
        lubridate::mdy() %>%
        error_on_date(date)
```

Reply:
Per our slack notes, I found that I didn't need to trim white space, otherwise, I implemented this suggestion as is, thank you!